include/triton/core/tritonbackend.h

```diff
@@ -411,6 +411,16 @@ TRITONBACKEND_ResponseFactoryIsCancelled(
 TRITONBACKEND_DECLSPEC TRITONSERVER_Error* TRITONBACKEND_RequestCorrelationId(
     TRITONBACKEND_Request* request, uint64_t* id);
 
+/// Extract the specified timeout value from the request. If no timeout is
```

Comment:
```suggestion
/// Get the timeout of the request. A value of 0 indicates no timeout was specified for the request.
```

Reply:
Trying to make a little more consistent with the description of other fields.
Can you also add a ticket to ensure the additional api is added to the bindings (Python / Java)

Reply:
Added DLIS-5869